modules/homa-lite/src/lib.rs

```diff
@@ -83,6 +90,20 @@ pub mod module {
 		/// The fixed cost of transaction fee for XCM transfers.
 		#[pallet::constant]
 		type MintFee: Get<Balance>;
+
+		/// The fixed cost of withdrawing Staking currency via redeem.
+		#[pallet::constant]
+		type BaseWithdrawFee: Get<Permill>;
+
+		/// TODO: ???
+		type Reserveldentifier: Get<ReserveIdentifier>;
```

Comment:
This is only needed for named reserves, which is not yet currently supported for MultiCurrency